plugin.audio.radiorecord/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.radiorecord" name="Radio Record" version="1.0.0+matrix.0" provider-name="Nemiroff">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0" />
+        <import addon="script.module.requests" version="2.4.3"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Radio Record</summary>
+        <summary lang="ru_RU">Радио Рекорд</summary>
+        <description lang="en_GB">RadioRecord.Ru in your Kodi!</description>
+        <description lang="ru_RU">Аддон для RadioRecord.Ru</description>
+        <disclaimer lang="en_GB">This is not official addon</disclaimer>
+        <disclaimer lang="ru_RU">Это не оффициальный аддон</disclaimer>
+        <language>ru en</language>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
Please update license to use SPDX identifier, THX
https://spdx.org/licenses/

Reply:
to `

Reply:
to `GNU General Public License v2.0 only` ?